docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cand_index = _select_candidates(subseqs, min_dist)
</pre><p>To be consistent with phase 2, this would be</p><p> </p><pre class="ql-syntax" spellcheck="false">cand_index = np.flatnonzero(_select_candidates(subseqs, min_dist))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Got your point. Thanks! Will be fixed in next commits.